R/utilities-sensitivity-analysis.R

```diff
@@ -24,6 +24,34 @@ runSensitivity <- function(simFilePath,
   # If there is a population file and no individualId then do SA for entire population
   # If there is no population file and individualId then do SA for mean model
   # If there is no population file and no individualId then do SA for mean model.
+
+  sim <- loadSimulation(simFilePath)
+
+  allVariableParameterPaths <- ospsuite::potentialVariableParameterPathsFor(simulation = sim)
+
+  if (is.null(variableParameterPaths)) { # If no parameters to vary specified, vary all parameters valid for sensitivity analysis

```

Comment:
Can you please put the comment above the text it's commenting not besides

Reply:
Also don't hesitate to put comment on multiple lines so that we do not have to scroll to read them

The goal is to avoid scrolling left to right as it disturbs the flow. Think 120 col ish max